cmd/payload-testing-prow-plugin/server.go

```diff
@@ -113,9 +114,30 @@ type trustedChecker interface {
 
 type githubTrustedChecker struct {
 	githubClient github.Client
+	trustedApps  prowflagutil.Strings
+}
+
+func (c *githubTrustedChecker) isTrustedApp(login string) bool {
+	if !strings.HasSuffix(login, "[bot]") {
```

Comment:
Is this consistent? Why could just pass the full name of the app in the args.

Reply:
ok. I updated this PR to remove this [bot] stripping logic and updated the release PR to include [bot] in the trusted app arg.